lib/l10n/intl_ru.arb

```diff
@@ -106,5 +106,27 @@
   "@download_to_device": {
     "type": "text",
     "placeholders": {}
+  },
+  "export_file" :"Export file",
+  "@export_file": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "preparing_to_export" :"Preparing to export",
+  "@preparing_to_export": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "cancel" :"Отменить",
+  "@cancel": {
+    "type": "text",
+    "placeholders": {}
+  },
+  "downloading_file": "Downloading {fileName}",
+  "@downloading_file": {
+    "type": "text",
+    "placeholders": {
+      "fileName": {}
+    }
```

Comment:
Create a ticket for each sprint to make all the text missing translation, we will request translate in pool. Thanks

Reply:
done: [https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/605](https://ci.linagora.com/linagora/lgs/linshare/products/linshare-mobile-apps/issues/605)